gradle/project.gradle.kts

```diff
@@ -21,18 +21,13 @@ import kotlin.text.Charsets.UTF_8
 
 // Note: the plugins block is evaluated before the script itself, so no variables can be used
 plugins {
-    id("com.gradle.build-scan") version "2.1"
+    id("com.gradle.build-scan") version "1.16"
     kotlin("jvm") version "1.3.0"
     application
     id("com.github.ben-manes.versions") version "0.20.0"
     id("de.undercouch.download") version "3.4.3"
 }
 
-buildScan {
```

Comment:
Was this not needed?

Reply:
It is from build-scan 2, because of the downgrade it resulted in errors. We will revert https://github.com/sghpjuikit/player/commit/445f905727ea207c1e7175464815142db01ffefa once everything works properly again.